src/content/blog/2021_unitaryhack_results.md

```diff
@@ -10,15 +10,15 @@ tags:
 
 #### We'd like to thank our sponsors [Xanadu](https://xanadu.ai/), [DoraHacks](https://dorahacks.com/), and [Pasqal](https://pasqal.io/) for helping to make this event a reality 💖
 
-At Unitary Fund, we help those interested in contributing to open quantum projects learn and practice the skills they need to do so. In May 2021, we organized [unitaryHACK](https://unitaryfund.github.io/unitaryhack/), a distributed quantum open source hackathon with 4k USD in bounties, physical and digital swag, and 20 participating projects. Working with open source project maintainers, we helped community members with a wide variety of backgrounds make meaningful contributions to their projects. Maintainers as well as Unitary Fund staff were on hand on our [Discord](http://discord.unitary.foundation) to help out with questions about the quantum technologies as well as the tools they use like git, GitHub, and a variety of languages and platforms used in the quantum space (Python, Julia, Q#, C++, Ruby, etc.)
+At Unitary Fund, we help those interested in contributing to open quantum projects learn and practice the skills they need to do so. In May 2021, we organized [unitaryHACK](https://unitaryfoundation.github.io/unitaryhack/), a distributed quantum open source hackathon with 4k USD in bounties, physical and digital swag, and 20 participating projects. Working with open source project maintainers, we helped community members with a wide variety of backgrounds make meaningful contributions to their projects. Maintainers as well as Unitary Fund staff were on hand on our [Discord](http://discord.unitary.foundation) to help out with questions about the quantum technologies as well as the tools they use like git, GitHub, and a variety of languages and platforms used in the quantum space (Python, Julia, Q#, C++, Ruby, etc.)
```

Comment:
worth mentioning here that `unitaryfoundation.github.io/unitaryhack` no longer works since we moved hosting to netlify. Not sure if we want to change this old links, but wanted to mention it.